### ISSUE TITLE
Can I customize the message text to save successfully?

### ISSUE THREAD

**Issue by hgaoping** [MarcinOrlowski/laravel-api-response-builder#112]
Can I customize the message text to save successfully?t here.

**Comment by MarcinOrlowski** [comment]
Yes, you can customize any string ResponseBuilder uses. It comes with defaults that (IMHO) make it not necessary, but if you want, there's `map` config directive that maps each code to specified message. See docs: https://github.com/MarcinOrlowski/laravel-api-response-builder/blob/master/docs/config.md#map

But if you want to pass any string from your code and have it used the apparently there's no variant of `success()` that accepts message string literally. I will add that in next  update.

**Comment by MarcinOrlowski** [comment]
There's new [successWithMessage()](https://github.com/MarcinOrlowski/laravel-api-response-builder/blob/dev/src/ResponseBuilder.php#L200) method added in dev branch. Please take a look if that's solves your problem.

**Comment by MarcinOrlowski** [comment]
Fixed in v6.3.2
